app/services/validators/common-validator.js

```diff
@@ -21,6 +21,10 @@ exports.isGreaterThanZero = function (value) {
   return value > 0
 }
 
+exports.isGreaterThanOrEqualToZero = function (value) {
+  return value >= 0
```

Comment:
The validator functions generally get chained, so most of the time, the isRequired validator (which checks for null/undefined etc) is called before any of the other validators.  I tested adding null checks to each of these validator functions and some of the unit tests have failed eg. isNumeric should throw an error if passed null.

I think adding null checks to these could cause issues with existing functionality so I am going to hold back on making this change for now.  Let me know what you think, and I can make the change later if needs be